onboarding/mentors.md

```diff
@@ -31,53 +71,83 @@ It's up to the mentoring pair to find what works best for them and to ensure the
+  - If they will be remote, where are they from? What is it like there & what works for them in terms of remote
+    work?
   - Are they new in town? Where do they live, for how long? Just have a normal conversation.
 - Prior experience with our [technology stacks](/playbooks/technology-choices.md) (and others)
 - Other relevant past experience, particularly 'soft skills' that might seem orthogonal to writing code
-- Are they using the [onboarding checklist](https://github.com/artsy/README/blob/master/.github/ISSUE_TEMPLATE/engineering-onboarding.md)? If so, check in on where they are and offer to be a resource on things there. Some tasks might stand out as good things to do in between People Ops meetings, while working nearby each other or to pair on.
+- Are they using the
+  [onboarding checklist](https://github.com/artsy/README/blob/master/.github/ISSUE_TEMPLATE/engineering-onboarding.md)?
+  If so, check in on where they are and offer to be a resource on things there. Some tasks might stand out as good
+  things to do in between People Ops meetings, while working nearby each other or to pair on.
+
+**Your first week**
 
-### Your first week
-- Check in to make sure your onboardee is moving forward in their checklist. Common stumbling areas include stale steps in the developer setup and getting invited to myriad organizations, lists and websites (slack, jira, github, notion, heroku, aws+hokusai, zeplin, ...)
+- Check in to make sure your onboardee is moving forward in their checklist. Common stumbling areas include stale
+  steps in the developer setup and getting invited to myriad organizations, lists and websites (slack, jira,
+  github, notion, heroku, aws+hokusai, zeplin, ...)
 - Look for chances to pair or work alongside one another throughout the week.
-  - If their early work hasn't exposed them to `artsy/gravity`, consider pairing on something trivial or just setting it up, breaking and fixing a test.
+  - If their early work hasn't exposed them to `artsy/gravity`, consider pairing on something trivial or just
+    setting it up, breaking and fixing a test.
   - If they haven't set up hokusai yet, consider pairing on that.
-- Try to have another meeting towards the end of the week- **especially** if they are leaving New York. Revisit things you talked about in the first meeting and look for areas that could have gone better. Consider amending relevant documents like this one.
+- Try to have another meeting towards the end of the week- **especially** if they are leaving New York. Revisit
+  things you talked about in the first meeting and look for areas that could have gone better. Consider amending
+  relevant documents like this one.
 
-### Your first year
-- New hires usually begin with a sprint-by-sprint rotation through our teams, a whirlwind that can last upwards of 2 months. It's good to check in with your mentee during this time.
+**Your first year**
```

Comment:
How about `The first few months`? Since this is primarily targeted at the sprint rotation period + the remainder of onboarding. (And we say above that it should last 3 months at minimum 😄 ).